2018/03/03 02:48:10 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2018/03/03 02:48:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in E:\Study\Yii\framework\db\CDbConnection.php:405
Stack trace:
#0 E:\Study\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 E:\Study\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 E:\Study\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 E:\Study\Yii\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 E:\Study\Yii\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 E:\Study\Yii\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 E:\Study\Yii\framework\db\ar\CActiveRecord.php(1585): CActiveRecord->getCommandBuilder()
#7 E:\Study\Yii\blog\protected\controllers\IndexController.php(24): CActiveRecord->findAllBySql('SELECT thumb,ai...')
#8 E:\Study\Yii\framework\web\actions\CInlineAction.php(49): IndexController->actionIndex()
#9 E:\Study\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\Study\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 E:\Study\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\Study\Yii\framework\web\CWebApplication.php(282): CController->run('')
#13 E:\Study\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#14 E:\Study\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 E:\Study\Yii\blog\index.php(16): CApplication->run()
#16 {main}
REQUEST_URI=/Study/Yii/blog/
HTTP_REFERER=http://127.0.0.1/Study/Yii/blog/
---
2018/03/03 02:48:10 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
